platform/linux/.gitconfig

```diff
@@ -0,0 +1,6 @@
+[credential]
+	helper = cache --timeout=172800
+[diff]
+	external = /home/smebberson/diff.sh
```

Comment:
@smebberson  This one still points to `smebberson`, I think the diff file is missing?

Reply:
Thanks. Fixed that.